auth-server/src/controllers/eventsHook.ts

```diff
@@ -126,6 +122,10 @@ const sendOwnProposalCreated = async (onchainLink): Promise<MessageType> => {
 		link = 'treasury';
 	}
 
+	if (onchain_motion_id === 0 || onchain_motion_id) {
```

Comment:
Motion still need to be **before** referendum, because a treasury can become a motion, but also a motion can become a referendum :)

so in this orfer proposal, treasury, motion, referendum. (whether proposal is before or after treasury is not important)

Reply:
ah i had this in mind but missed

Reply:
done